app/views/environments/welcome.html.erb

```diff
@@ -4,11 +4,10 @@
     <%= icon_text("th", "", :kind => "fa") %>
   </div>
   <h1><%= _('Environments') %></h1>
-  <p><%= _('If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.') %></p>
-  <%= _('This information is commonly imported from a pre-existing puppet configuration by the use of the') %> <%= link_to  _('Puppet classes and environment importer'), documentation_url("4.2.2Classes"), :rel => "external" %> <%= _('Puppet classes and environment importer') %></a></br>
-  <%= _('Environments may be manually created and only require the name of the environment to be declared.') %></br>
+  <p><%= _("If you are planning to use Foreman as an external node classifier you should provide information about one or more environments. This information is commonly imported from a pre-existing puppet configuration by the use of the <a href='#{documentation_url("4.2.2Classes")}' rel=external>Puppet classes and environment importer</a>.").html_safe %>
```

Comment:
You can't use `#{}` string interpolation inside an extracted string, it won't extract or translate properly (because it'll interpolate _first_ and then be looked up in the translations - so won't be found).  Interpolation has to be done _after_ translation, e.g. `_('Foo %s baz') % 'bar'`.  The documentation URL should be interpolated this way.

It also contains HTML, which we should avoid - see 19df8e10 for one way of doing it (link_start/end).

The original string is bad too, because it concatenates two sentence fragments: "by use of the" [..] "Puppet classes and environment importer".  This makes it hard to translate, because translators don't see the two strings next to each other, or know that one follows the other.  Putting it in one string as you've done is an improvement.

Lastly, if you're changing the string, fix the capitalisation of "Puppet".